packages/veritone-react-common/src/components/SourceManagement/NullState/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import withMuiThemeProvider from 'helpers/withMuiThemeProvider';
+import Button from '@material-ui/core/Button';
+import { func } from 'prop-types';
+
+import NullstateImage from 'images/cms-sources-null.svg';
+import styles from './styles.scss';
+
+@withMuiThemeProvider
+export default class SourceManagementNullState extends React.Component {
+  static propTypes = {
+    onClick: func
```

Comment:
This should be required in my opinion, unless you can foresee a use case where a user will render a button that doesn't do anything?